proof/invariant-abstract/DetSchedInvs_AI.thy

```diff
@@ -581,26 +581,30 @@ end
 
 \<comment> \<open>Project threads from the kernel heap\<close>
 
-definition tcb_of :: "kernel_object \<rightharpoonup> tcb" where
```

Comment:
See my other comment for this -- we should leave it as `definition` with `case`, and use `[split_simps ..]` to get the simp rules.